examples/simple-grep/README.md

```diff
@@ -219,15 +206,15 @@ occurencesPrinter = mapM_ unpack
             printLine line
 
         printIdxWithColon :: Int -> App ()
-        printIdxWithColon idx = Iris.putStderrColouredLn
+        printIdxWithColon idx = Iris.putStderrColoured
           (Colourista.formatWith [Colourista.yellow, Colourista.bold])
-            $ BSL.toStrict $ (TLE.encodeUtf8 $ T.pack $ " " `mappend` show idx)
+            $ TS.pack $ " " `mappend` show idx
               `mappend`
-              (TLE.encodeUtf8 $ T.pack ":")
+              ":"
         printLine :: T.Text -> App ()
-        printLine x = Iris.putStdoutColouredLn
-            (Colourista.formatWith [BS.empty])
-                $ BSL.toStrict $ TLE.encodeUtf8 x
+        printLine x = Iris.putStdoutColoured
+            (Colourista.formatWith [TS.empty])
+            $ T.toStrict $ x `mappend` "\n"
```

Comment:
There I used `putStdoutColoured` without `Ln` for stable purposes. When the API of `putStdoutColouredLn` will be changed from `ByteString` to `Text`, there could be a need to change `simple-grep` too. So, I just used \n